backend/test/resolvers/entry.js

```diff
@@ -1,21 +1,58 @@
 import { expect } from 'chai'
 
 import Entry from '../../models/entry'
+import Group from '../../models/group'
 import { createPhoto, createVideo, createOtherMedia } from '../../resolvers/mutations/entry'
 import { fakeUser, fakeShow } from '../factories'
 
+const standardEntry = (user, show) => ({
```

Comment:
Should we create a factory for Entry?

Reply:
probably not necessary here, since the point of testing this method is to create the Entry object itself